mantidproject/mantid
# Re #20728 Added a new except statement to catch runtime error

Description of work.

**To test:**
Download the ISIS sample data.
OPen sans GUI v2
select MaskFile.txt as the relevant mask file
type 74044 as the sample scatter run
hit process

The expected result is that this will raise an exception saying that it cannot find the file. 


Fixes #20728  

**Release Notes** 
Bug fix so no release notes needed.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
